docs/dev/api_usage_examples/get_listens.py

```diff
@@ -0,0 +1,59 @@
+'''
+get_listens.py
+'''
+
+from common import AUTH_HEADER, USERNAME, Track, construct_url
+
+import requests as req
+
+# The token in AUTH-HEADER must be valid, but it doesn't have to be the token of the user you're
+# trying to get the listen history of.
+def get_listens(username, min_ts=None, max_ts=None, count=None):
+    """Gets the listen history of a given user.
+
+    Args:
+        username: User to get listen history of.
+        min_ts: History before this timestamp will not be returned.
+                DO NOT USE WITH max_ts.
+        max_ts: History after this timestamp will not be returned.
+                DO NOT USE WITH min_ts.
+        count: How many listens to return. If not specified,
+               uses a default from the server.
+
+    Returns:
+        A list of listen info dictionaries if there's an OK status.
+
+    Raises:
+        An HTTPError if there's a failure.
+        A ValueError if the JSON in the response is invalid.
+        An IndexError if the JSON is not structured as expected.
+    """
+    url = construct_url(f"user/{username}/listens")
+
+    # Thanks to requests, if any of these values are None,
+    # they're not going to be added in the first place.
+    params = {
+        "min_ts": min_ts,
+        "max_ts": max_ts,
+        "count": count
+    }
+
+    response = req.get(url, params=params, headers=AUTH_HEADER)
+
+    response.raise_for_status()
+
+    return response.json()['payload']['listens']
+
+if __name__ == "__main__":
+    listens = get_listens(USERNAME)
+    listens_tracks = [Track(track_dict) for track_dict in listens]
+
+    for track in listens_tracks:
+        print(track.simple_repr())
+
+def verify_args(username, min_ts=None, max_ts=None, count=None):
```

Comment:
This does not seem to be relevant to the api endpoints much. Maybe it can be removed.

Reply:
Yeah, even I wasn't sure about these - I'll remove both the `verify_args` functions.